osf-cas/templates/configmap.yaml

```diff
@@ -386,6 +380,37 @@ config/cas.properties: |-
   cas.logout.remove-descendant-tickets=false
   ########################################################################################################################
 
+  ########################################################################################################################
+  # OSF URLs
+  ########################################################################################################################
+  # OSF
+  #
+  cas.authn.osf-url.home=https://{{ .Values.osfDomain }}/
+  cas.authn.osf-url.dashboard=https://{{ .Values.osfDomain }}/dashboard/
+  cas.authn.osf-url.login-with-next=https://{{ .Values.osfDomain }}/login?next=
+  cas.authn.osf-url.logout=https://{{ .Values.osfDomain }}/logout/
+  cas.authn.osf-url.resend-confirmation=https://{{ .Values.osfDomain }}/resend/
+  cas.authn.osf-url.forgot-password=https://{{ .Values.osfDomain }}/forgotpassword/
+  cas.authn.osf-url.register=https://{{ .Values.osfDomain }}/register/
+  ########################################################################################################################
```

Comment:
Current stating3 newCAS has these URLs hard coded in the `messages.properties`. With new backend support added along with institution SSO, newCAS now loads these domain-dependent URLs from `cas.properties`.